src/libmscore/layout.cpp

```diff
@@ -1700,10 +1700,10 @@ static void distributeStaves(Page* page)
                 Spacer* activeSpacer { nextSpacer };
                 nextSpacer = nullptr;
                 for (MeasureBase* mb : system->measures()) {
-                    Measure* m = toMeasure(mb);
-                    if (!mb->isMeasure()) {
+                    if (!(mb && mb->isMeasure())) {
                         continue;
                     }
+                    Measure* m = toMeasure(mb);
```

Comment:
Maybe is it better to add a check on null?

Reply:
That's done already in line 1703. `toMeasure()` is actually just a cast from `Element` to `Measure` and the test two lines ahead make sure `m` isn't a `nullptr`.